components/dashboard/src/index.css

```diff
@@ -16,6 +16,9 @@
         /* Setup RGB color channel variables */
         --black: 22 22 22; /* #161616 */
         --white: 255 255 255; /* #FFFFFF */
+        /* TODO: determine if these are the reds we want - need to have correct constrast */
```

Comment:
Needed some red colors so set this up for now. The red in the podkit designs didn't have the right contrast so went with these for now (from tailwind reds).